src/webapp/js/components/locations.js

```diff
@@ -25,24 +25,38 @@ const CHIEFDOM_COLUMNS = [
   },
 ];
 
-const FACILITIES_COLUMNS = [
-  {
-    Header: 'Facility ID',
-    accessor: 'facilityId',
-  }, {
-    Header: 'Name',
-    accessor: 'name',
-  }, {
-    Header: 'Facility Type',
-    accessor: 'facilityType',
-  }, {
-    Header: 'Incharge name',
-    accessor: 'inchargeFullName',
-  }, {
-    Header: 'Chiefdom',
-    accessor: 'parent',
-  },
-];
+class FACILITIES_COLUMNS extends Component {
```

Comment:
Why this is made as Component? I dont get the idea behind it.

Reply:
@pstudzinski revert this change and just add the filter

Reply:
I Split Facility Type Filter to new const, removed Component.